java/cdk-environment/src/test/java/sleeper/environment/cdk/buildec2/KeyPairUtilIT.java

```diff
@@ -15,89 +15,85 @@
  */
 package sleeper.environment.cdk.buildec2;
 
-import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
 
 import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.net.URL;
-import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.nio.file.attribute.PosixFilePermission;
 import java.security.KeyPair;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 class KeyPairUtilIT {
+    @TempDir
+    private Path tempDir;
+
+    @BeforeEach
+    public void setup() throws IOException {
+        KeyPair pair = KeyPairUtil.generate();
+        Files.writeString(tempDir.resolve("private.pem"), KeyPairUtil.privatePem(pair));
+        Files.writeString(tempDir.resolve("public.base64"), KeyPairUtil.publicBase64(pair));
```

Comment:
I can't tell which tests use which parts of this setup. Could the necessary parts be moved to the relevant tests?